tests/scene/test_emotions.py

```
@@ -439,7 +440,7 @@ def test_descriptions_same_dates():
 def test_descriptions_no_dates():
     personA = Person(name="A")
     personB = Person(name="B")
-    conflict = Emotion(personA, personB, kind=util.ITEM_CONFLICT)
+    conflict = Emotion(RelationshipKind.Conflict, personA, personB)
     assert conflict.startEvent.description() == None
```

Ruff (0.13.3)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)